Examples/DataRepresentation/Image/ImageToArray.py

```diff
@@ -14,32 +14,33 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 #
-#==========================================================================*/
+# ==========================================================================*/
+
+from sys import argv
 
 from InsightToolkit import *
 from numarray import *
-from sys import argv
 
 reader = itkImageFileReaderUC2_New()
 
 connector = itkPyBufferUC2_New()
 
-reader.SetFileName( argv[1] )
+reader.SetFileName(argv[1])
 
 reader.Update()
 
-print "ready to convert image into array"
+print
+"ready to convert image into array"
```

Comment:
Is this Python2 or Python3 style?

Reply:
In commit message pychar**n** - > pychar**m**

Reply:
It's Python 2 code, since there's no parens around the string.  I'm not sure why pycharm split the line like that, since line length wasn't an issue.

Reply:
fixed pycharn typo

Reply:
Aren't we moving away from Python2? I think we already officially dropped support.